### PR TITLE
Added Missing Dependency Suspected to be Removed after Dependabot Update

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: testing/e2e/skr/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@kubernetes/client-node": "0.22.1",
     "axios": "^1.7.4",
+    "execa": "^9.4.0",
```
Comment (jaroslaw-pieszka) [inline testing/e2e/skr/package.json:31]: image

Comment (jaroslaw-pieszka) [reply testing/e2e/skr/package.json:31]: During dependabot update this dependence was not removed

Comment (jaroslaw-pieszka) [reply testing/e2e/skr/package.json:31]: Since the last dependabot update was reverted this PR should be closed.

Comment (ralikio) [reply testing/e2e/skr/package.json:31]: That was not the missing dependency.

**Pull request closed without merging**